test/objectFifo-stateful-transform/bd_chain_on_memtile/infinite_loop.mlir

```diff
@@ -0,0 +1,59 @@
+//===- bd_chain_on_memtile/infinite_loop.mlir ----------------*- MLIR -*-===//
```

Comment:
I think I may be misunderstanding but just to confirm, isn't the infinite loop the standard behavior? If that is the case I don't think you need to add a test for that since it is captured in the base tests of the objectfifo.